groovy/postInit/electronics/EtchablesChain.groovy

```diff
@@ -147,9 +133,6 @@ class NDopant {
 
 //FEEL FREE TO ADD MORE IF YOU FIND THEM
 
-new Photoresist("novolacs", 50, 300)
```

Comment:
tbh you can probably revert the changes in this file because we can just make the EUV multiblock exclusively use PMMA instead of making PMMA only usable in the EUV multi